egs/librispeech/WSASR/conformer_ctc2/train.py

```diff
@@ -373,6 +382,9 @@ def get_params() -> AttributeDict:
 
         - warm_step: The warm_step for Noam optimizer.
     """
+    parser = get_parser()
+    LibriSpeechAsrDataModule.add_arguments(parser)
+    args = parser.parse_args()
```

Comment:
```suggestion
```

Reply:
I delete this part but i have one question. how should feature-dim argument read from input??

Reply:
I delete this part but i have one question. how should feature-dim argument read from input??

@csukuangfj